components/infobox/wikis/starcraft/infobox_person_player_custom.lua

```diff
@@ -0,0 +1,500 @@
+---
+-- @Liquipedia
+-- wiki=starcraft
+-- page=Module:Infobox/Person/Player/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local CleanRace = require('Module:CleanRace')
+local Info = require('Module:Info')
+local Json = require('Module:Json')
+local Logic = require('Module:Logic')
+local Lpdb = require('Module:Lpdb')
+local Math = require('Module:Math')
+local Namespace = require('Module:Namespace')
+local Notability = require('Module:Notability')
+local Person = require('Module:Infobox/Person')
+local RaceIcon = require('Module:RaceIcon')
+local String = require('Module:StringUtils')
+local Table = require('Module:Table')
+local Variables = require('Module:Variables')
+
+local Condition = require('Module:Condition')
+
+local ConditionTree = Condition.Tree
+local ConditionNode = Condition.Node
+local Comparator = Condition.Comparator
+local BooleanOperator = Condition.BooleanOperator
+local ColumnName = Condition.ColumnName
+
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+
+local _CURRENT_YEAR = tonumber(os.date('%Y'))
+local _ALLOWED_PLACES = {'1', '2', '3', '4', '3-4'}
+local _EARNING_MODES = {['solo'] = '1v1', ['team'] = 'team'}
+local _MAXIMUM_NUMBER_OF_PLAYERS_IN_PLACEMENTS = 30
```

Comment:
Number doesn't match one of the other infoboxes.
Should it be moved to a common module?

Reply:
yeah we can probably add it to info module (tomorrow)